app/models/comment.php

```diff
@@ -20,17 +20,116 @@ public static function getComments($offset, $limit, $id)
         $db = DB::conn();
         $rows = $db->rows("SELECT * FROM comment 
                         WHERE thread_id = ? ORDER BY created ASC LIMIT {$offset}, {$limit}", array($id));
```

Comment:
It will treat the value as string if placeholders are put for $offset and $limit :bow: